src/game/logic/object/locomotor.cpp

```diff
@@ -1853,3 +1908,295 @@ void Locomotor::Start_Move()
 {
     m_moveFrame = (g_theGameLogic->Get_Frame() + 2.5f * 30.0f);
 }
+
+bool Locomotor::Loco_Update_Maintain_Current_Position(Object *obj)
+{
+    if (!Get_Flag(MAINTAIN_POS_IS_VALID)) {
+        m_maintainPos = *obj->Get_Position();
+        Set_Flag(MAINTAIN_POS_IS_VALID, true);
+    }
+
+    m_moveFrame = g_theGameLogic->Get_Frame() + 2.5f * 30.0f;
```

Comment:
is this same as startmove?

Reply:
I don't know but I can confirm its 100% not an inlined copy of Start_Move.